addons/mail/static/src/core/common/thread.xml

```diff
@@ -53,7 +57,7 @@
             </div>
         </t>
         <t t-else="">
-            <div class="o-mail-Thread-empty d-flex flex-column align-items-center justify-content-center p-4 text-muted fst-italic h-100">
+            <div class="o-mail-Thread-empty d-flex flex-column align-items-center justify-content-center text-muted fst-italic h-100" t-att-class="{'p-4': !props.closeThread}">
```

Comment:
Again if you want to customise showing/hiding empty text, make a dedicate prop for that